neonvm-kernel/linux-config-amd64-6.6.64

```diff
@@ -631,24 +656,32 @@ CONFIG_KVM_COMPAT=y
 CONFIG_HAVE_KVM_IRQ_BYPASS=y
 CONFIG_HAVE_KVM_NO_POLL=y
 CONFIG_KVM_XFER_TO_GUEST_WORK=y
+CONFIG_KVM_GENERIC_HARDWARE_ENABLING=y
 CONFIG_VIRTUALIZATION=y
 CONFIG_KVM=y
 CONFIG_KVM_WERROR=y
 # CONFIG_KVM_INTEL is not set
 # CONFIG_KVM_AMD is not set
+CONFIG_KVM_SMM=y
 # CONFIG_KVM_XEN is not set
+# CONFIG_KVM_PROVE_MMU is not set
 CONFIG_AS_AVX512=y
 CONFIG_AS_SHA1_NI=y
 CONFIG_AS_SHA256_NI=y
 CONFIG_AS_TPAUSE=y
+CONFIG_AS_GFNI=y
+CONFIG_AS_WRUSS=y
 CONFIG_ARCH_CONFIGURES_CPU_MITIGATIONS=y
 
 #
 # General architecture-dependent options
 #
-CONFIG_CRASH_CORE=y
-CONFIG_KEXEC_CORE=y
 CONFIG_HOTPLUG_SMT=y
```

Comment:
For later, we should probably disable this after switching to sysfs scaling. cc @mikhail-sakhnov